src/components/VirtualizedTreeSelect.js

```diff
@@ -190,11 +209,19 @@ class VirtualizedTreeSelect extends Component {
     if ("onOptionToggle" in this.props) {
```

Comment:
I suggest removing this if, we know for certian that `onOptionToggle` is in props (see `render` in `IntelligentTreeSelect`).

Reply:
Yes, you are correct, it is not necessary to check it. 
Removed